src/pages/en/reference/integrations-reference.md

```diff
@@ -266,37 +280,80 @@ The address, family and port number supplied by the [NodeJS Net module](https://
 
 ### `astro:build:done`
 
-**Previous hook:** [astro:build:ssr](#astrobuildssr)
+**Previous hook:** [`astro:build:ssr`](#astrobuildssr)
 
 **When:** After a production build (SSG or SSR) has completed.
+
 **Why:** To access generated routes and assets for extension (ex. copy content into the generated `/assets` directory). If you plan to transform generated assets, we recommend exploring the [Vite Plugin API](https://vitejs.dev/guide/api-plugin.html) and [configuring via `astro:config:setup`](#updateconfig-option) instead.
 
 ```js
-'astro:build:done'?: (options: { pages: { pathname: string }[]; dir: URL; routes: RouteData[] }) => void | Promise<void>;
```

Comment:
`pages` is essentially `routes` with a lot less information. This property is only used by our `sitemap` integration currently, while `routes` is used by everything else. I'm planning to deprecate and remove this from the core codebase for v1 to avoid confusion!